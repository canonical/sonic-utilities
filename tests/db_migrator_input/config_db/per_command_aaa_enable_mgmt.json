{
    "AAA|authentication": {
        "login": "tacacs+"
    },
    "TACPLUS|global": {
        "auth_type": "login",
        "passkey": "testpasskey"
    },
    "DEVICE_METADATA|localhost": {
        "type": "MgmtToRRouter"
    }
}