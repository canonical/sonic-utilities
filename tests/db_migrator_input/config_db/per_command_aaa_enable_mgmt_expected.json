{
    "AAA|accounting": {
        "login": "tacacs+,local"
    },
    "AAA|authentication": {
        "login": "tacacs+"
    },
    "AAA|authorization": {
        "login": "tacacs+,local"
    },
    "TACPLUS|global": {
        "auth_type": "login",
        "passkey": "testpasskey"
    },
    "DEVICE_METADATA|localhost": {
        "type": "MgmtToRRouter"
    }
}